resources/todomvc/big-dom-generator/index.js

```diff
@@ -2,16 +2,22 @@ import React from "react";
 import { renderToStaticMarkup } from "react-dom/server";
 import { App } from "./src/app";
 import { genCss } from "./gen-css";
+import { genShadowDomVariables } from "./gen-shadow-dom-variables";
 
+const path = require("path");
 const fs = require("fs");
```

Comment:
Convert these to imports.
```suggestion
import path from "path";
import fs from "fs";
```